src/components/z-cover-hero/styles.css

```diff
@@ -0,0 +1,81 @@
+:host {
+  --cover-hero-height: auto;
+  --cover-hero-overlay: none;
+  --cover-hero-text-color: var(--color-text-inverse);
+
+  display: block;
+}
+
+:host,
+* {
+  box-sizing: border-box;
+}
+
+:host .content-hero {
+  position: relative;
+  align-items: baseline;
+  width: 100%;
+  height: var(--cover-hero-height);
+  background-color: var(--gray600);
+}
+
+:host .content-container {
+  color: var(--cover-hero-text-color);
+  fill: var(--cover-hero-text-color);
+}
+
+/* Container of the slotted cover */
+:host .content-hero .cover {
+  position: absolute;
+  z-index: 0;
+  width: 100%;
+  height: 100%;
+}
+
+::slotted([slot="cover"]) {
+  position: absolute;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
+  object-fit: cover;
+  object-position: top center;
+}
+
+::slotted(z-info-reveal) {
+  position: absolute;
+  bottom: var(--grid-margin);
+  right: var(--grid-margin);
+  z-index: 1;
```

Comment:
se siamo in variante overlay i due `z-index: 1` vanno in conflitto e non si apre più la didascalia

Reply:
risolto :)